src/Translations/TranslationIntegrationModel.cs

```diff
@@ -133,7 +133,7 @@ public class TranslationIntegrationModel
         public MicroUserModel LastUpdatedByUser { get; set; }
 
         /// <summary>
-        /// Gets or sets a value indicating whether this <see cref="TranslationIntegration"/> is active.
+        /// Gets or sets a value indicating whether this <see cref="TranslationIntegrationModel"/> is active.
```

Comment:
Not sure if that <see cref> is needed here, it's referencing to the same model anyways.

Reply:
Do you care if I remove the entire "<see/>" so it doesn't cause a stylecop error?